src/pages/admin/regulatory/CreateRegulatory.jsx

```diff
@@ -119,8 +117,20 @@ class CreateRegulatory extends React.Component{
   }
 
   render(){
-    const{classes, client}  = this.props;
-    const initialValue = this.state.client;
+    const{classes}  = this.props;
```

Comment:
You need to pass an initial value to formik. Ideally, this should be similar to your Yup schema